docs/docs/using-gatsby-image.md

```diff
@@ -37,25 +37,67 @@ With Gatsby, we can make images way way better.
 
 `gatsby-image` is designed to work seamlessly with Gatsby’s native image processing capabilities powered by GraphQL and Sharp. To produce perfect images, you only need to:
 
-1. Import gatsby-image and use it in place of the built-in img
-1. Write a GraphQL query using one of the included GraphQL “fragments” which specify the fields needed by `gatsby-image`.
+1. Install `gatsby-image` and other, necessary dependencies like `gatsby-plugin-sharp` and `gatsby-transformer-sharp`
 
-The GraphQL query creates multiple thumbnails with optimized JPEG and PNG compression. The `gatsby-image` component automatically sets up the “blur-up” effect as well as lazy loading of images further down the screen.
+```shell
+  npm i gatsby-image gatsby-transformer-sharp gatsby-plugin-sharp
+```
+
+2. Add the newly installed plugins and transformer plugins to your `gatsby-config.js`
 
-Here’s what a really simple Gatsby page component using gatsby-image would look like:
+```js:title=gatsby-config.js
+module.exports = {
+  plugins: [`gatsby-plugin-sharp`, `gatsby-transformer-sharp`],
+}
+```
+
+3. Configure `gatsby-source-filesystem` to load images from a folder. In order to use GraphQL to query the image files, the files need to be in a location that is known to Gatsby. This requires an to `gatsby-config.js` to configure the plugin. Feel free to replace the `path` option with wherever your images are located relative to your project.
```

Comment:
>This requires an to `gatsby-config.js`...

I think this needs a word like `entry` or `edit` here, i.e. "This requires an edit to `gatsby-config.js...`"